src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -886,6 +892,47 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly nonQueriedMolecularData = remoteData<NumericGeneMolecularData[]>({
+        await: () => [
+            this.studyToDataQueryFilter,
+            this.genes,
+            this.nonSelectedMolecularProfiles,
+            this.samples
+        ],
+        invoke: () => {
+            // we get mutations with mutations endpoint, all other alterations with this one, so filter out mutation genetic profile
+            const profilesWithoutMutationProfile = _.filter(this.nonSelectedMolecularProfiles.result, (profile: MolecularProfile) => profile.molecularAlterationType !== 'MUTATION_EXTENDED');
```

Comment:
i believe there's an enum for somewhere for MUTATION_EXTENDED.  see if you can find it and use it.